bee-crypto/src/ternary/hash.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+use bee_ternary::{Trits, T1B1};
+
+use std::{cmp::PartialEq, fmt, hash};
+
+/// The length of a hash in units of balanced trits.
+pub const HASH_LENGTH: usize = 243;
+
+/// Ternary cryptographic hash.
+#[derive(Copy, Clone)]
```

Comment:
Would love to see derive Serialize and Deserialize trait here.  Or maybe add a feature flag for it like what bee-ternary does.

Reply:
`Debug` would likely be quite useful too

Reply:
`Hash` is `Debug`

Reply:
If I remember correctly we did not implement Serialize and Deserialize since of different encodings?

Reply:
We'll consider Serialize and Deserialize in a coming version. In the meantime you'll be able to get the inner trits which are Serialize and Deserialize.